packages/eslint-config-eslint/eslintrc.js

```diff
@@ -0,0 +1,451 @@
+/**
+ * TODO: the config will be removed in the future, please use the index config.
+ * @deprecated
+ * @fileoverview the eslintrc config - it's exported as some eslint's tests still rely on it.
```

Comment:
Are there still any tests that rely on this? I'd keep this file for some time for IDEs anyway, but if there are tests that still need this project's configs it would be good to update them.

Reply:
I've checked it's no longer used by tests, pushed a commit https://github.com/eslint/eslint/pull/17247/commits/906b58f6d48185a724e2d2fc51df1f3d51431ca6 to clarify.